### PR TITLE
fix: add acceptHeader to browserInfo

### PR DESCRIPTION
**Pull request by descorp**
### Changes

* add `acceptHeader` to BrowserInfo

### CODE REVIEW

File: Adyen/Model/BrowserInfo.swift [hunk 1]
```diff
@@ -12,6 +12,9 @@ public struct BrowserInfo: Encodable {
     
     /// The device default user-agent.
     public var userAgent: String?
+
+    /// The Accept request HTTP header indicates which content types, expressed as MIME types, the client is able to understand.
+    public var acceptHeader: String { "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8" }
```
Comment (mohammedDehairy) [inline Adyen/Model/BrowserInfo.swift:17]: why is this a computed property?

also where did you get this value?

**Comment by mohammedDehairy** [comment]
spell check is failing because of `html,application` :-), you can add it to `.typokana_ignore` file as an exception.

File: Adyen/Model/BrowserInfo.swift (hunk 1 above)
Comment (mohammedDehairy) [inline Adyen/Model/BrowserInfo.swift:17]: spell check is failing because of `html,application` :-), you can add it to `.typokana_ignore` file as an exception.

File: Adyen/Model/BrowserInfo.swift [hunk 2]
```diff
@@ -32,10 +35,21 @@ public struct BrowserInfo: Encodable {
             completion(BrowserInfo(userAgent: cachedUserAgent))
         }
     }
+
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        try container.encode(acceptHeader, forKey: .acceptHeader)
+        try container.encodeIfPresent(userAgent, forKey: .userAgent)
+    }
```
Comment (mohammedDehairy) [inline Adyen/Model/BrowserInfo.swift:43]: why is this needed? the default implementation should work, right?

File: Adyen/Model/BrowserInfo.swift (hunk 1 above)
Comment (descorp) [reply Adyen/Model/BrowserInfo.swift:17]: 👍

File: Adyen/Model/BrowserInfo.swift (hunk 2 above)
Comment (descorp) [reply Adyen/Model/BrowserInfo.swift:43]: Same as [previous](https://github.com/Adyen/adyen-ios/pull/713#discussion_r754027972) question:
For some reason it crashes 🤷‍♂️
